setup-env/action.yml

```diff
@@ -0,0 +1,21 @@
+name: 'Setup Environment'
+description: 'Setup dependencies for base container'
+
+runs:
+  using: "composite"
+  steps:
+    - name: Install updates and missing packages
+      run: |
+        export DEBIAN_FRONTEND=noninteractive
+        apt update && apt upgrade -y
+
+        # Install git if not exist
+        if ! command -v git &> /dev/null; then
+          apt install git -y
+        fi
+
+        # Install awscli if not exist
+        if ! command -v aws &> /dev/null; then
+          apt install awscli -y
+        fi

```

Comment:
Do we need to install other stuff like npm and node? Or is that already installed as part of the Docker image?

Reply:
It is been taken care of by other actions, e.g: https://github.com/Brightspace/bdp-batch/blob/US116893/githubAction/.github/workflows/ci.yml#L23